src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlFieldMonacoEditorSettings.Edit.cshtml

```diff
@@ -3,59 +3,29 @@
 @{ 
     var urlPrefix = String.IsNullOrWhiteSpace(shellSettings.RequestUrlPrefix) ? string.Empty : String.Concat(shellSettings.RequestUrlPrefix, '/');
     var IStandaloneEditorConstructionOptionsSchemaPath = $"/{urlPrefix}OrchardCore.Resources/Scripts/monaco/IStandaloneEditorConstructionOptions.json";
+    var diagnosticsOptions = System.Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(string.Format(@"{{
```

Comment:
Using base64 here as I could not get the JSON.parse() on the client to read the output of the json encoder.

Reply:
Maybe change this to  `JsonConvert.SerializeObject`